package.json

```diff
@@ -4,9 +4,8 @@
   "description": "Custom elements (web components) for making audio and video player controls that look great in your website or app.",
   "main": "dist/index.js",
   "files": [
-    "./dist/*",
-    "README.md",
-    "./examples/*"
```

Comment:
I don't think examples/ needs to be published with the package, right?

Reply:
It's actually nice to be able to pull them up directly on the CDN.